core/storage/email/gae_models.py

```diff
@@ -148,11 +150,11 @@ def get_by_hash(cls, email_hash, sent_datetime_lower_bound=None):
                 sent_datetime of the email to be searched.
 
         Returns:
-            A list of messages.The hashed messages with the given hash value
-                and sent_datemtime greater than lower bound.
+            list(str). A list of emails which have the given hash value and 
```

Comment:
it returns the list of all the instances of SentEmailModel which has given hash value. The id of instances of this class has the form intent. Intent is in form str.